src/jdk.httpserver/share/classes/com/sun/net/httpserver/Filter.java

```diff
@@ -47,39 +51,41 @@
     protected Filter () {}
 
     /**
-     * a chain of filters associated with a HttpServer.
-     * Each filter in the chain is given one of these
-     * so it can invoke the next filter in the chain
+     * A chain of filters associated with a {@link HttpServer}.
+     * Each filter in the chain is given one of these so it can invoke the
+     * next filter in the chain.
      */
     public static class Chain {
-        /* the last element in the chain must invoke the users
-         * handler
+
+        /**
+         * The last element in the chain must invoke the users
```

Comment:
invoke the users -> invoke the user's

Reply:
Change made in commit https://github.com/openjdk/jdk/pull/429/commits/4de7e6cac8681914f94c407b032fb8c1cb172828